packages/core/src/App/Containers/Modals/exit-traders-hub-modal/exit-traders-hub-modal.jsx

```diff
@@ -30,7 +32,7 @@ const ExitTradersHubModal = ({
                 <Modal.Body>{exit_traders_hub_modal_content}</Modal.Body>
                 <Modal.Footer has_separator>
                     <Button onClick={closeModal} has_effect text={localize('Not now')} secondary large />
-                    <Button has_effect onClick={onClickExitButton} text={localize('Yes, exit')} primary large />
+                    <Button has_effect onClick={onClickExitButton()} text={localize('Yes, exit')} primary large />
```

Comment:
This change shouldn't make a difference. `()` is already applied on function call when it's used on `onClick()`